### ISSUE TITLE
Atom crashes on startup

### ISSUE THREAD

**Issue by denysdovhan** [atom/atom#10520]
I've no idea why, but after reboot of my system, when I try to run Atom, it just show me this and nothing more:

![image](https://cloud.githubusercontent.com/assets/3459374/12452868/2daf17da-bf99-11e5-9b75-d7d491eb3f53.png)

I'm working on **Ubuntu GNOME 15.10**. Error:

```
SyntaxError: Unexpected end of input
    at Object.parse (native)
    at FileSystemBlobStore.load (/opt/atom/resources/app.asar/src/file-system-blob-store.js:31:31)
    at Function.load (/opt/atom/resources/app.asar/src/file-system-blob-store.js:10:14)
    at window.onload (file:///opt/atom/resources/app.asar/static/index.js:18:39)
```

**Comment by denysdovhan** [comment]
I've just removed my `~/.atom` folder and it started up as well. But I still dunno what was the cause of this. I hadn't installed anything and hadn't changed its configurations.

**Comment by Zireael07** [comment]
It says there was an error in BlobStore.load, so I guess your blob-store got corrupted somehow.

**Comment by cameronbarker** [comment]
Had the same issue when I loaded up this morning.  Removing ~/.atom resolved my issue as well.  Thanks @denysdovhan

**Comment by denysdovhan** [comment]
@cameronbarker thanks.

Looks like removing of `~/.atom/blob-store` was enough to make it works again.

**Comment by 50Wliu** [comment]
This is already being looked into (I can't find the original issue right now unfortunately).

**Comment by denysdovhan** [comment]
@50Wliu please give us reference to original issues. Personally, I wanna get updates about fixing.

**Comment by 50Wliu** [comment]
@denysdovhan Found it (it has a generic title which is why I couldn't when I briefly searched around for it): #10015.  It will be fixed by #10293.

**Comment by denysdovhan** [comment]
@50Wliu ok, thanks.
